maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ProceedingsMapper.java

```diff
@@ -28,8 +29,16 @@ public class ProceedingsMapper extends CrownCourtMapper {
     public ApiUpdateApplicationRequest workflowRequestToUpdateApplicationRequest(
             ApplicationDTO application, UserDTO userDTO) {
 
-        ApiUpdateApplicationRequest updateApplicationRequest = new ApiUpdateApplicationRequest()
-                .withRepId(NumberUtils.toInteger(application.getRepId()))
+        ApiUpdateApplicationRequest request = new ApiUpdateApplicationRequest();
+        mapToApiProcessRepOrderRequest(application, request, Boolean.TRUE);
```

Comment:
We don't call the `preProcessRequest` endpoint, so may as well combine these two methods.